src/components/login/LoginComponent.js

```diff
@@ -0,0 +1,134 @@
+import react,{useRef,useState,useEffect} from'react';
+import * as FaIcons from "react-icons/fa";
+// import * as yup from 'yup';
+import * as EmailValidator from 'email-validator';
+import {Formik} from 'formik'; 
+import { Link,useNavigate } from 'react-router-dom';
+
+// import PersonIcon from '@mui/icons-material/Person';
+import "./Login.css";
+import { toast, ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+
+const LoginComponent = () => {
+  const [users,setUsers] = useState(null);
+  const [loading,setIsLoading] = useState(false);
+  const redirect = useNavigate();
+  useEffect(()=>{
+    fetch("http://localhost:3003/user")
+    .then((res)=>{
+      return res.json();
+    })
+    .then ((data)=>{
+  setUsers(data)
+  setIsLoading(false)
+    })
+  })
+
+  const handleLogin = (values)=>{
+   setIsLoading(true)
+   
+
+    users.map((user)=>{
+      
+ if((user.email === values.email) && (user.password === values.password)){
+   toast.info("User logiged in successfully ")
+   
+   localStorage.setItem("token","toke33nphantddomfaketokensdgdgss")
+   redirect("/dashboard")
```

Comment:
The value of token in localStorage must be the token from the backend response. Please remove the hardcoded token.